crates/apps/src/lib/cli/utils.rs

```diff
@@ -154,6 +160,41 @@ pub const fn arg_multi<T, K>(name: &'static str) -> ArgMulti<T, K> {
     }
 }
 
+pub trait WrappedHelp {
+    fn wrapped_help(self, help: &str) -> Self;
+    #[allow(dead_code)]
+    fn wrapped_long_help(self, help: &str) -> Self;
+}
+
+impl WrappedHelp for ClapArg {
+    fn wrapped_help(self, help: &str) -> Self {
+        let dictionary = Standard::from_embedded(Language::EnglishUS).unwrap();
+        let options = Options::new(LINE_WRAPPING)
+            .word_splitter(WordSplitter::Hyphenation(dictionary));
```

Comment:
maybe I'd also make these `options` a `LazyStatic` global, because I've noticed a small decrease in printing speed of the help msgs, which might partially be due to this being constructed for every flag (though I'm unsure if it does any transformations on the data structures at this stage)

Reply:
It's probably the fact that it is recomputing the wrapping every time maybe? If I use macros, I can make the whole string formatting happen at compile time. I'll make a reference implementation and we'll see if we like it.

Reply:
yeah, compile time formatting would solve this I guess. try to use `konst`. we have it in scope in `apps` I think

Reply:
there were problems trying to use that unless I wanted to implement line wrapping from scratch. But I was able to use a macros version of the same crate. The first time the help is accessed, it's noticably slower, but every time after that it's not.

Reply:
I must admit I don't like the way it looks as much though.

Reply:
I reran the modified code. the help msgs do indeed print much faster; in fact, now they print instantly (or feel like that anyway), whereas before they took like 100ms to fully load. cosmetically, I don't notice any difference on the help msgs themselves. the code looks a bit uglier for sure, but I prefer better ux over better dx.